mono/utils/mono-dl-windows.c

```diff
@@ -50,20 +51,22 @@ mono_dl_open_file (const char *file, int flags)
 	if (file) {
 		gunichar2* file_utf16 = g_utf8_to_utf16 (file, strlen (file), NULL, NULL, NULL);
 
-#if G_HAVE_API_SUPPORT(HAVE_CLASSIC_WINAPI_SUPPORT)
+#ifdef HAVE_API_SUPPORT_WIN32_SET_ERROR_MODE
```

Comment:
Should be `if` not `ifdef` so that it can be explicitly stated as not present, instead of ambiguous omission.